tests/core/full_node/stores/test_block_store.py

```diff
@@ -52,7 +54,24 @@ async def test_block_store(
         pytest.skip("only run in PLAIN mode to save time")
 
     assert sqlite3.threadsafety >= 1
-    blocks = bt.get_consecutive_blocks(10)
+
+    blocks = bt.get_consecutive_blocks(
+        3,
+        guarantee_transaction_block=True,
+        farmer_reward_puzzle_hash=bt.pool_ph,
+        pool_reward_puzzle_hash=bt.pool_ph,
+        time_per_block=10,
+    )
+    wt: WalletTool = bt.get_pool_wallet_tool()
+    tx: SpendBundle = wt.generate_signed_transaction(
+        uint64(10), wt.get_new_puzzlehash(), list(blocks[-1].get_included_reward_coins())[0]
+    )
```

Comment:
without this, none of the test blocks would have a block generator, and the tests below would not cover some interesting cases